clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -3583,24 +3585,44 @@ void populateCIRToLLVMConversionPatterns(mlir::RewritePatternSet &patterns,
 }
 
 namespace {
+
+using IRASMap = mlir::cir::AddressSpaceAttr::MapTy;
+
+/// Returns the CIR address space map when available. Returns nullptr if the
+/// triple is not present in the module.
+IRASMap const *getIRAddrSpaceMap(mlir::ModuleOp module) {
+  // If the triple is not present, e.g. CIR modules parsed from text, we
+  // cannot init LowerModule properly.
+  assert(!::cir::MissingFeatures::makeTripleAlwaysPresent());
+  // Here we will use a default AS map to ignore all AS stuff.
+  if (!module->hasAttr("cir.triple"))
```

Comment:
What testcases related to address space break right now when we don't have `cir.triple`? We should either update them or place a direct assert so that folks in the future don't right tests like that.